src/runner/PlutoRunner/src/PlutoRunner.jl

```diff
@@ -1350,7 +1350,7 @@ end
 function show_richest_withreturned(context::IOContext, @nospecialize(args))
     buffer = IOBuffer(; sizehint=0)
     val = show_richest(IOContext(buffer, context), args)
-    return (resize!(buffer.data, buffer.size), val)
```

Comment:
This function should return the final `Vector{UInt8}`, not a view, because it will be sent to the Julia server and then to JS.

You could wrap it in `collect`, but I was hoping to avoid the allocations. The performance is important because this is part of the display and messaging pipeline. E.g. when you move a slider that controls a Plots.jl plot, this function gets called on the plot SVG data.

Reply:
`buffer.data` is of now of type `Memory{UInt8}`, which has no `resize!` method. See the "Memory PR" commit.
https://github.com/JuliaLang/julia/commit/909bceae8a60969dfceba4924142e5efee459d47

Not sure what do do here.  `return resize (Vector(buffer.data),buffer.size),val)` seems to be possible, but likewise probably would allocate. 
Didn't look up all the discussions on the Memory stuff, so I don't know if lacking `resize!` is due to concept or due to omission.

Reply:
Fixed in https://github.com/fonsp/Pluto.jl/pull/2828 👍